scripts/update_all_products.pl

```diff
@@ -1323,22 +1323,8 @@
 
 			# Otherwise, we silently update the .sto file of the last version
 			else {
-
-				# make sure nutrient values are numbers
-				ProductOpener::Products::make_sure_numbers_are_stored_as_numbers($product_ref);
```

Comment:
I think you removed ProductOpener::Products::make_sure_numbers_are_stored_as_numbers($product_ref); from the script, but did not add it to the function.